iOS/Controllers/SettingsViewController.swift

```diff
@@ -65,14 +69,24 @@ class SettingsViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let selectedCell = tableView.cellForRow(at: indexPath)
-        if selectedCell == logoutCell {
+        guard let selectedCell = tableView.cellForRow(at: indexPath) else { return }
+
+        switch selectedCell {
+        case logoutCell:
             LoginHelper.logout()
             let loginViewController = R.storyboard.main.login()!
             if #available(iOS 13, *) {
                 loginViewController.modalPresentationStyle = .fullScreen
             }
             self.present(loginViewController, animated: true, completion: nil)
+        case imprintCell:
+            let safariViewController = SFSafariViewController(url: Brand.default.servers.imprint)
+            self.present(safariViewController, animated: true)
+        case dataUsageCell:
+            let safariViewController = SFSafariViewController(url: Brand.default.servers.dataPrivacy)
+            self.present(safariViewController, animated: true)
```

Comment:
Please also set the `tintColor` of the control buttons in the `SFSafariViewController`.
https://developer.apple.com/documentation/safariservices/sfsafariviewcontroller/2274393-preferredcontroltintcolor